docs/contributors/code/native-mobile.md

```diff
@@ -12,10 +16,6 @@ Also, the mobile client is packaged and released via the [official WordPress app
 
 The majority of the code shared with native mobile is in the very same JavaScript module and SASS style files. In the cases where the code paths need to diverge, a `.native.js` or `.native.scss` variant of the file is created. In some cases, platform specific files can be also found for Android (`.android.js`) or iOS (`.ios.js`).
```

Comment:
I would move this `Native mobile specific files` section to be the second section because it seems to me that this is information that is going to be important for more people compared to actually running Android and iOS. wdyt?

Reply:
I agree, I think it's better that the first information we display explains how the mobile related code fits in the project, this gives a quick overall vision. The instructions for running and testing should go afterwards. 

I applied this change in this [commit](https://github.com/WordPress/gutenberg/pull/30195/commits/f01c9821bab2b3ac595967dd894b25accc06f70f).